pkg/addons/translate.go

```diff
@@ -543,25 +543,13 @@ var Translations = map[string][]struct {
 	"Route.route.openshift.io/default/docker-registry": {
 		{
 			Path:     jsonpath.MustCompile("$.spec.host"),
-			Template: "docker-registry-default.{{ (index .ContainerService.Properties.RouterProfiles 0).PublicSubdomain }}",
+			Template: "docker-registry.{{ (index .ContainerService.Properties.RouterProfiles 0).PublicSubdomain }}",
 		},
 	},
 	"Route.route.openshift.io/default/registry-console": {
 		{
 			Path:     jsonpath.MustCompile("$.spec.host"),
-			Template: "registry-console-default.{{ (index .ContainerService.Properties.RouterProfiles 0).PublicSubdomain }}",
-		},
-	},
-	"Route.route.openshift.io/kube-service-catalog/apiserver": {
-		{
-			Path:     jsonpath.MustCompile("$.spec.host"),
-			Template: "apiserver-kube-service-catalog.{{ (index .ContainerService.Properties.RouterProfiles 0).PublicSubdomain }}",
-		},
-	},
-	"Route.route.openshift.io/openshift-ansible-service-broker/asb-1338": {
-		{
-			Path:     jsonpath.MustCompile("$.spec.host"),
-			Template: "asb-1338-openshift-ansible-service-broker.{{ (index .ContainerService.Properties.RouterProfiles 0).PublicSubdomain }}",
```

Comment:
i am not sure what effect this has..

Reply:
I think it might be an issue if somebody wants to integrate with asb from outside. Needs testing if asb does not use it internally. (just to be sure)

Reply:
I was able to deploy an APB.  AFAIK an external user should use the service catalog API via the main apiserver, not the broker API.